### PR TITLE
Update rapids-cmake functions to non-deprecated signatures

### PR DESCRIPTION
**Pull request by robertmaynard**
Update to use non deprecated signatures for `rapids_export` functions

### CODE REVIEW

**Review by bdice — APPROVED** [review]

**Comment by robertmaynard** [comment]
@bdice @madsbk Any reason why I can't merge this? We will need this to unblock 23.04 builds

**Comment by bdice** [comment]
I agree with your assessment and went ahead and merged.

**Pull request merged**
